googlemock/include/gmock/gmock-generated-actions.h

```diff
@@ -1,4 +1,6 @@
-// This file was GENERATED by a script.  DO NOT EDIT BY HAND!!!
+// This file was GENERATED by command:
```

Comment:
It would be best to keep the formatting so the changes are limited to the actual purpose

Reply:
Hi,

I do not understand this request. I did the "real" change in `.../gmock-generated-actions.h.pump`, and then I regenerated the `.h` file with:
```sh
$ ./googletest/scripts/pump.py googlemock/include/gmock/gmock-generated-actions.h.pump
```
Any diff in the generated `.h` file is either result of (other) changes in `pump.py` or due to my changes in `.../gmock-generated-actions.h.pump`. Inspection of the main function in `pump.py` indeed shows that pump.py is responsible for the changed output.

The reason I regenerated the header file is twofold:
* There is no Cmake support for generating the header files.
* To show that dynamic template types where correctly generated (this seems to have been misleading however).

It seems that the root cause of the unwanted change in the header file(s) is that they where not regenerated when pump.py was changed. The optimal solution would be to generate the header files from Cmake, so that error prone manual steps are eliminated. I could have easily done this in make, but unfortunately I have never written CMake scripts

Please advice if I should keep the re-generated header file (i.e. not update the pull request) or revert the generation and just keep the change in the `.pump` file?

Reply:
@Osram-Lux Sorry for the confusion, you are correct. There is no need to invest in making pump changes in CMake/make at this point.